nusight2/src/client/components/robot_selector/style.css

```diff
@@ -69,5 +69,5 @@
     0px 0px 3px 0px rgba(0, 0, 0, 0.15),
     0px 3px 8px 0px rgba(0, 0, 0, 0.15);
   overflow: hidden;
-  width: 100%;
+  width: 330px;
```

Comment:
It seems strange to me to select an absolute width. Will this break scaling on some monitors?

Reply:
Potentially, although I think most monitors that you will be viewing NUsight on will be larger than 330px. The issue is a total width of 100% will automatically wrap to the smallest it can be, whereas adding this width absolutely means that it will render with some padding that will look more readable

Reply:
This panel overlays over the existing display of NUsight so it will not be pushing any other elements out of the way

Reply:
seems reasonable enough